rpilocator-rss-global.py

```diff
@@ -0,0 +1,222 @@
+import requests
+import feedparser
+import time
+import json
+import os
+from dotenv import load_dotenv
+from pathlib import Path
+
+load_dotenv()
+
+# String to Boolean
+def str2bool(str):
+    return str.lower() in ("yes", "true", "t", "1")
+
+# Variable loader
+def getvar(variable_name, default=''):
+    var = os.getenv(variable_name)
+    filevar = variable_name + '_FILE'
+    filevarpath = Path('/run/secrets/' + filevar)
+    filevarcontent = ''
+    
+    if filevarpath.exists():
+        with open(filevarpath, 'r') as f:
+            filevarcontent = f.readline().strip()
```

Comment:
The pattern is usually to be able to name the file (secret) whatever you want, not necessarily the name of the corresponding ENV. So something like this would check if for example `PUSHOVER_KEY_FILE` is set, read that value and use that as the path of the file. The value of `PUSHOVER_KEY_FILE` could be something like `/var/run/my_extra_special_secret`. 

```suggestion
    filevar = os.getenv(variable_name + '_FILE')
    filevarcontent = ''
    if filevar:
        filevarpath = Path(filevar)    
        if filevarpath.exists():
            with open(filevarpath, 'r') as f:
                filevarcontent = f.readline().strip()
```

Reply:
Right, that makes sense. I never used this method for variables, so tell me if you find anything like this